src/components/Progress/ProgressItem.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { IconButton, ButtonIcon } from '../Button';
+import { Icon } from '../Icon';
+
+const ProgressItem = ({
+  assistiveText,
+  isActive,
+  isError,
+  isCompleted,
+  isVertical,
+  children,
+  isSuccess,
```

Comment:
By convention, we spread the `...rest` props to public components (`className` would have to be merged into cx). You can ignore this here if we decide to implement an array prop as explained above.